checker/tests/nullness/RawTypesNullness18.java

```diff
@@ -0,0 +1,32 @@
+// @below-java18-jdk-skip-test
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Generic18<G extends @Nullable Object> {}
+
+class MyClass18 extends Generic18<MyClass18> {}
+
+class BoundedGeneric18<B extends @Nullable CharSequence> {}
+
+class RawTypesNullness18 {
+  Generic18 rawReturn() {
+    return new Generic18();
+  }
+
+  Generic18 rawField = new Generic18();
+}
+
+class TestBounded18 {
+  BoundedGeneric18 rawReturn() {
+    return new BoundedGeneric18<>();
+  }
+
+  BoundedGeneric18 rawField = new BoundedGeneric18();
+
+  void useBoundedWildCard() {
+    BoundedGeneric18 rawLocal = new BoundedGeneric18<String>();
+    BoundedGeneric18<? extends Object> generic1 = rawReturn();
```

Comment:
So the Checker Framework is going to have different defaulting rules for  `? extends Object` wildcards depending on the Java version? Or is this something we can fix later?

Reply:
It should not.  Which of these is correct?

Reply:
Oh I just noticed that the warnings in RawTypesNullness17.java are issued by `javac` not by the Checker Framework work.  You could add a `@SuppressWarnings("unchecked")` to this file and it would pass with all version of Java.  The Checker Framework passed both versions.